openreferee_server/server.py

```diff
@@ -44,6 +45,10 @@
 )
 
 
+def get_notifier(context) -> NotifyService:
+    return context.extensions.get('notifier')
```

Comment:
You `.get()` it but if it's `None`, calling `notify()` on it will fail. so use `context.extensions['notifier']` here or handle the case where `get_notifier()` returns `None` below